include/mega/logging.h

```diff
@@ -199,66 +431,48 @@ class SimpleLogger {
         SimpleLogger::logCurrentLevel = ll;
     }
 
+    // set whether we're in performance mode (default: false)
+    static void setPeformanceMode(bool enable)
+    {
+        SimpleLogger::performanceMode = enable;
+    }
+
+    // are we in performance mode?
+    static bool inPerformanceMode()
+    {
+        return SimpleLogger::performanceMode;
+    }
+
     // Synchronizes all registered stream buffers with their controlled output sequence
     static void flush();
 };
 
-// output VERBOSE log with line break
 #define LOG_verbose \
-    if (SimpleLogger::logCurrentLevel < logMax) ;\
+    if (::mega::SimpleLogger::logCurrentLevel < ::mega::logMax) ;\
```

Comment:
Added explicit namespace here to ensure that one can call these macros without issuing a `using namespace mega` first.